### PR TITLE
ug: offline-update: Adjustments in return codes documentation

### PR DESCRIPTION
**Pull request by detsch**
# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 8c3414320b27f5b1e3299fbe8492200f20b50513 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2661/docs/artifacts/html/index.html

**Review by mike-scott — APPROVED** [review]
LGTM

**Review by mike-sul — APPROVED** [review]

File: source/user-guide/offline-update/offline-update.rst [hunk 1]
```diff
@@ -161,6 +161,8 @@ The ``install`` command sets the following exit codes:
     - Reboot is required to complete installation. After reboot ``aklite-offline run`` must be invoked.
 - *101*: Installation succeeded.
     - Restart of dockerd service is required to complete installation, e.g. ``systemctl restart docker``. After the restart ``aklite-offline run`` must be invoked.
+- *102*: Downgrade attempt.
+    - The Target to be installed has a version lower than the one that is running. The ``--force`` option can be used to force its installation.
```
Comment (kprosise) [inline source/user-guide/offline-update/offline-update.rst:165]:
```suggestion
    - The Target to be installed has a version lower than the one running. Use the ``--force`` option to force its installation.
```

**Review by kprosise — APPROVED** [review]
One optional suggestion, otherwise it LGTM.

File: source/user-guide/offline-update/offline-update.rst (hunk 1 above)
Comment (detsch) [reply source/user-guide/offline-update/offline-update.rst:163]: Thanks, I've applied the suggested change.

**Comment by doanac** [comment]
Docs for ec552b2995847a46453c13a6c3d82eee4cfc13c0 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2664/docs/artifacts/html/index.html

**Comment by detsch** [comment]
@kprosise @mike-scott  I've added two additional small changes to the PR

**Review by kprosise — APPROVED** [review]
LGTM

**Review by mike-sul — APPROVED** [review]

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**